.env.example

```diff
@@ -6,9 +6,12 @@ EMAIL_SENDER=<transactional-email-sender>
 EMAIL_VERIFY_SECRET_TOKEN=whatever_you_want_this_to_be_it_only_matters_for_production
 EXAMPLE_USER_EMAIL=examples@p5js.org
 EXAMPLE_USER_PASSWORD=hellop5js
-GG_EXAMPLES_EMAIL=benedikt.gross@generative-gestaltung.de
-GG_EXAMPLES_PASS=generativedesign
```

Comment:
btw this is NOT the password in production, and maybe it makes sense to leave just the username here, and put in a dummy email/pw, same for the ml5 account

Reply:
Hi @catarak - hooray! Welcome back from your trip + thank for reviewing :) 

> this looks great to me! the only thing is, you didn't add any changes to packages in package.json, so i think it makes sense to remove the package-lock.json from this PR.
+ Oops! I will remove the package-lock.json from this PR. Is it the case that I can just `git rm --cache package-lock.json` ?

> btw this is NOT the password in production, and maybe it makes sense to leave just the username here, and put in a dummy email/pw, same for the ml5 account
+ Ah, cool. Sounds good. I can add in a dummy placeholder for ml5

Reply:
to remove `package-lock.json` it's probably easiest to just make a new commit, since you have a lot of commits on this PR and i don't know which one has the `package-lock` changes. so you'll have to do, from your feature branch,
`$ git checkout master -- package-lock.json`
`$ git add .`
`$ git commit -m "reset package-lock.json to master"`

Reply:
Oh wow! I didn't realize it was possible to cherry pick files like this! 🤯